cadre.go

```diff
@@ -44,27 +45,27 @@ type cadre struct {
 
 func (c *cadre) Start() error {
 	sigs := make(chan os.Signal, 1)
-	sigsDone := make(chan bool)
+	c.sigsDone = make(chan bool)
```

Comment:
move to the top

Reply:
sry, i meant that this should be moved to the cadre's construction, i.e.: Builder's `Build` function.